### DISCUSSION TITLE
Second monitor?

### DISCUSSION THREAD

**Discussion by dellmelo** [zhongyang219/TrafficMonitor#1574] [General]
Hello, first of all... congratulations for the great software. Love it.

I have two monitors and I´d like to see my hw info when I´m in fullscreen on the primary monitor.

How can I add TrafficMonitor info on second monitor?

Thanks.

**Comment by caematos** [comment]
I have the same question

**Comment by ltzJack** [comment]
Same here

**Comment by atpersian** [comment]
I have the same question
